src/Repository/RepositoryInterface.php

```diff
@@ -14,6 +14,15 @@
     public function find($id);
 
     /**
+     * Find multiple objects by identifier
```

Comment:
@elazar Small change, but maybe change to "Find multiple objects by multiple identifiers"